aiohttp_jwt/utils.py

```diff
@@ -1,10 +1,9 @@
 import asyncio
-import re
 
 
 def check_request(request, entries):
     for pattern in entries:
-        if re.match(pattern, request.path):
+        if pattern in request:
```

Comment:
here can be complex pattern used for whitelisting routes based on regex not only part of string. If so it may make sense to check if pattern just a string or regex and decide what to use but i barely sure it will make any valuable performance boost.